package.json

```diff
@@ -27,5 +50,13 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@storybook/addon-actions": "^5.3.8",
+    "@storybook/addon-links": "^5.3.8",
+    "@storybook/addons": "^5.3.8",
+    "@storybook/preset-create-react-app": "^1.5.2",
+    "@storybook/react": "^5.3.8",
+    "eslint-plugin-react-hooks": "^4.0.8"
```

Comment:
Are we making of the eslint, because I see many syntax error that lint would not allow before merging.